docs/conf.py

```diff
@@ -67,9 +67,11 @@ def get_tutorial_images():
     sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), ".."))
     from mslib.mswms.demodata import DataFiles
 
-    root_fs = fs.open_fs("~/")
-    if not root_fs.exists("mss/testdata"):
-        root_fs.makedirs("mss/testdata")
+    with fs.open_fs("~/") as root_fs:
+        if not root_fs.exists("mss/testdata"):
+            root_fs.makedirs("mss/testdata")
+        if not root_fs.exists(".config/msui"):
+            root_fs.makedirs(".config/msui")
```

Comment:
there should be already an issue for this by @joernu76  for $XDG_CONFIG_HOME.  On first attempt we had problems by that. But  maybe these are now solved.

App Filesystems can make a point for Pyfilesystem, sometimes I think if we should deprecate it.

Reply:
I think you mean #908.

Reply:
> sometimes I think if we should deprecate it.

You mean remove fs entirely?

I think we should either fully remove it, or go all-in on fs. The current state of mixing everything everywhere is a mess and results in practically nothing but OSFS working, which makes fs somewhat pointless.

Not sure what would be better. I like python's builtin pathlib module, as an alternative to os, and haven't really seen a need for fs yet.

Reply:
I moved that to the next major, we should collect pro's and con's first. 

https://github.com/Open-MSS/MSS/issues/2103

When we introduced FS we had anything using it. Over the years it gots a mixup. At the time we introduced it there was no pathlib and we also had problems with tests on windows because of path. The idea that we use it with FS pathes seems not become the standard. We can use it with our nextcloud but we don't. etc.
Also with mscolab we can do things differently, e.g. exchange images, or config over chat.